src/main/java/frc/trigon/robot/commands/LEDAutoSetup/LEDAutoSetupCommand.java

```diff
@@ -0,0 +1,67 @@
+package frc.trigon.robot.commands.LEDAutoSetup;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.trigon.robot.RobotContainer;
+import frc.trigon.robot.subsystems.ledstrip.LEDStrip;
+import frc.trigon.robot.subsystems.ledstrip.LEDStripCommands;
+import frc.trigon.robot.subsystems.ledstrip.LEDStripConstants;
+
+import java.util.function.Supplier;
+
+public class LEDAutoSetupCommand extends ParallelCommandGroup {
+    private static final double TOLERANCE_METERS = 0.02;
+    private final Pose2d targetPose;
+    private final LEDStrip
+            leftStrip = LEDStripConstants.LEFT_STRIP,
+            rightStrip = LEDStripConstants.RIGHT_STRIP;
+
+    public LEDAutoSetupCommand(Supplier<Pose2d> targetPose) {
```

Comment:
actually, make this accept a string supplier for auto name, and generate the pose using `PathPlannerAuto.getStaringPoseFromAutoFile()` 
Also, don't generate the target pose on code start, that defeats the purpose of it being a supplier...